### PR TITLE
fix(vcl): revert caching directive for dispatch

### LINKED ISSUE #424: Rollback/Disable #413 (cache in production)?
#413 caches responses in the inner CDN I an outer CDN is making the request.

As we don't have an automated invalidation mechanism yet this means that some things are cached very stubbornly and you need to flush both inner and outer CDN for changes to become effective. 

To make matters wors^H even more interesting, the cache is varied by XFH, so that a given URL might be cached as 200 *and* 404 response.

As I've been burned by this (and as @davidnuescheler made sure to share his pain), I suggest we temporarily roll back this feature (after the go-live) until @tripodsan's solution for invalidating Helix Pages upon updates from Google, MS and GitHub is in place. 

This will create more load for us, especially for tracking URLs (#402), but that would be only a temporary issue.

### PR DESCRIPTION
**Pull request by tripodsan**
fixes #424

needs https://github.com/adobe/helix-dispatch/pull/329 to be in place

### CODE REVIEW

**Comment by kptdobe** [comment]
Why did you remove @trieloff from the reviewers ? His review is more useful than mine, I cannot judge if the change is good or not ;)

File: vcl/extensions.vcl [hunk 1]
```diff
@@ -11,14 +11,8 @@
 #
 
 sub hlx_type_pipeline_before {
-    # A non-production XFH header looks like: "repo--user.hlx.page"
-    # If you have an outer CDN, then it looks like "www.mydomain.com, repo--user.hlx.page"
-    # The filter below tests if you have no outer CDN and turns on the development mode
-    # caching (i.e. disables caching in helix-dispatch)
-    if(regsuball(regsuball(req.http.x-forwarded-host, "[^,]*(\.hlx(-[0-9]+)?\.page|\.project\-helix\.page|\.fastlydemo\.net)", ""), "[, ]", "") == "") {
```
Comment (trieloff) [inline vcl/extensions.vcl:18]: I'd rather add a `&& false` so that we can re-enable it easily.

**Review by trieloff — CHANGES_REQUESTED** [review]
Let's. Are it easier to toggle between the two modes.

File: vcl/extensions.vcl (hunk 1 above)
Comment (tripodsan) [reply vcl/extensions.vcl:18]: I can just comment everything :-)

**Comment by tripodsan** [comment]
let's just keep it in....

**Comment by tripodsan** [comment]
> Why did you remove @trieloff from the reviewers ? His review is more useful than mine, I cannot judge if the change is good or not ;)

I think this was an accident..... I would never remove @trieloff from any caching-related PR :-)

**Review by trieloff — APPROVED** [review]

**Pull request merged**